src/js/api/posts/getListOfListings.js

```diff
@@ -0,0 +1,20 @@
+// Author: Emilie Herrera Thomsen
+// The Dummy json needs to be replaced with actual endpoint.
+
+import { authBaseFetchOpen } from "../apiBaseFetch.js";
+
+export async function getListOfListings() {
+    const getListingsUrl = "https://dummyjson.com/products";
+
+    const response = await authBaseFetchOpen(getListingsUrl);
```

Comment:
Im not sure if I'm writing the test the correct way, but I get a response that says "fetch is not defined in authBaseFetchOpen"